packages/docusaurus-plugin-content-blog/src/plugin-content-blog.d.ts

```diff
@@ -180,7 +190,14 @@ yarn workspace v1.22.19image` is a collocated image path, this entry will be the
     | BlogPostFrontMatterAuthor
     | (string | BlogPostFrontMatterAuthor)[];
 
-  export type BlogPostMetadata = {
+  export type LastUpdateData = {
```

Comment:
can be moved to docusaurus utils alongside the method that returns that data structure